src/Operations.jl

```diff
@@ -1282,6 +1282,7 @@ function gen_test_code(testfile::String;
         --color=$(Base.have_color === nothing ? "auto" : Base.have_color ? "yes" : "no")
         --compiled-modules=$(Bool(Base.JLOptions().use_compiled_modules) ? "yes" : "no")
         --check-bounds=yes
+        --depwarn=$(Base.JLOptions().depwarn == 2 ? "error" : "yes")
```

Comment:
If I have started Julia explicitly with `--depwarn=no` it would be nice to propagate that too, but maybe that info is not available as for other options?

Reply:
Doesn't seem like it is available. Guess you need to just pass it explicitly.

Reply:
Yep there's not much we can do about this without considerably complicating `Base.JLOptions()` with the ability to distinguish between default vs explicitly specified options.